documentation/contributor_doc.md

```diff
@@ -0,0 +1,57 @@
+# Nexus Replication Plugin
+
+[https://github.com/griddynamics/nexus-replication-plugin](https://github.com/griddynamics/nexus-replication-plugin)
+
+This is Nexus plugin that provides replication logic when new (Maven) artifacts are stored in a hosted repository. 
```

Comment:
Sounds debatable to me. I would say that the logic of the replication is provided by the plugin - of course some stuff is done by Nexus Core, but not the replication itself.. 

Or at least I'd say that it _facilitates_ the replication.